File: src/main/java/yeh/pratice/Hello.java
package yeh.pratice;

public class Hello {
    public static void main(String[] args) {

        new Person().Hello();
        int i=4;
        char c='L';
        byte b=127;
        float f=12.7f;
        double d=12.7;
        long l=4L;
        boolean B=true;
        String s="小麻糬";
//        alt+9
//        ctrl+d
//        alt+`

    }

    static class Person{
        void Hello(){
            System.out.println("Hello Java");
        }
    }
}

File: src/main/kotlin/yeh/pratice/Hello.kt
package yeh.pratice

fun main() {
//    println("Hello");
//    Person().hello();
    val h=Person();
    h.hello();
//    val 不可改變
//    var 可改變
//    var  age:Int =123;
    var age=123;
    var kg:Double=65.5;
    var name:String;
    name="小麻糬";
}

class Person{
    fun hello(){
        println("hello kotlin");
    }
}